script/lib/util.py

```diff
@@ -258,3 +259,12 @@ def get_buildtools_executable(name):
   if sys.platform == 'win32':
     path += '.exe'
   return path
+
+def get_objcopy_path(target_cpu):
+  if PLATFORM == 'linux' and (target_cpu == 'x64' or target_cpu == 'x'):
```

Comment:
what's `'x'`? this doesn't make sense as a target cpu. if you're receiving `x` here it's because something earlier in the pipeline is messed up.

Reply:
Ops it is a typo it should have been empty string instead, this is because in the config.yml, there is a step-maybe-electron-dist-strip which set the target_cpu as x64 if the var TARGET_ARCH is empty on linux, should I do the same ?

Reply:
Done